.github/workflows/merge-commits-to-next-branch.yml

```
@@ -29,72 +29,73 @@ jobs:
             const queueClient = queueServiceClient.getQueueClient(queueName);
             var receivedMessages = await queueClient.receiveMessages();
             const message = receivedMessages.receivedMessageItems[0];
-            $COMMIT_INFO = JSON.parse(message.messageText);
+            const firstMessage = JSON.parse(message.messageText);
+            return firstMessage;
+    outputs:
+      LABEL: ${{ fromJSON(steps.deque-message.outputs.result).label }}
+      AUTHOR: ${{ fromJSON(steps.deque-message.outputs.result).author }}
+      SHA: ${{ fromJSON(steps.deque-message.outputs.result).sha }}
   create-branch:
     runs-on: ubuntu-latest
-    if: ( ($COMMIT_INFO).label == 'queued' )
+    permissions:
+      contents: write
+      packages: write
```

try

```suggestion
    permissions:
      actions: write
      checks: write
      contents: write
      issues: write
      pull-requests: write
      statuses: write
```

And if that doesn't work try `permissions: write-all` just for the job that pushes the HEAD.

---

write-all didn't work on my fork. I'll try adding all permissions